grasa_event_locator/templates/editEvent.html

```diff
@@ -75,80 +75,80 @@ <h5 class="modal-title" id="exampleModalLabel">Submit</h5>
+                                <option value="NC"{% ifequal address.2.strip "NC" %} selected{% endifequal %}>NC</option>
+                                <option value="NE"{% ifequal address.2.strip "NE" %} selected{% endifequal %}>NE</option>
+                                <option value="NH"{% ifequal address.2.strip "NH" %} selected{% endifequal %}>NH</option>
+                                <option value="NJ"{% ifequal address.2.strip "NJ" %} selected{% endifequal %}>NJ</option>
+                                <option value="NM"{% ifequal address.2.strip "NM" %} selected{% endifequal %}>NM</option>
+                                <option value="NV"{% ifequal address.2.strip "NV" %} selected{% endifequal %}>NV</option>
+                                <option value="NY"{% ifequal address.2.strip "NY" %} selected{% endifequal %}>NY</option>
+                                <option value="ND"{% ifequal address.2.strip "ND" %} selected{% endifequal %}>ND</option>
+                                <option value="OH"{% ifequal address.2.strip "OH" %} selected{% endifequal %}>OH</option>
+                                <option value="OK"{% ifequal address.2.strip "OK" %} selected{% endifequal %}>OK</option>
+                                <option value="OR"{% ifequal address.2.strip "OR" %} selected{% endifequal %}>OR</option>
+                                <option value="PA"{% ifequal address.2.strip "PA" %} selected{% endifequal %}>PA</option>
+                                <option value="RI"{% ifequal address.2.strip "RI" %} selected{% endifequal %}>RI</option>
+                                <option value="SC"{% ifequal address.2.strip "SC" %} selected{% endifequal %}>SC</option>
+                                <option value="SD"{% ifequal address.2.strip "SD" %} selected{% endifequal %}>SD</option>
+                                <option value="TN"{% ifequal address.2.strip "TN" %} selected{% endifequal %}>TN</option>
+                                <option value="TX"{% ifequal address.2.strip "TX" %} selected{% endifequal %}>TX</option>
+                                <option value="UT"{% ifequal address.2.strip "UT" %} selected{% endifequal %}>UT</option>
+                                <option value="VT"{% ifequal address.2.strip "VT" %} selected{% endifequal %}>VT</option>
+                                <option value="VA"{% ifequal address.2.strip "VA" %} selected{% endifequal %}>VA</option>
+                                <option value="WA"{% ifequal address.2.strip "WA" %} selected{% endifequal %}>WA</option>
+                                <option value="WI"{% ifequal address.2.strip "WI" %} selected{% endifequal %}>WI</option>
+                                <option value="WV"{% ifequal address.2.strip "WV" %} selected{% endifequal %}>WV</option>
+                                <option value="WY"{% ifequal address.2.strip "WY" %} selected{% endifequal %}>WY</option>
```

Comment:
Is there validation if a user types in `New York` for a state instead of `NY`? @nlMeminger @leong96 @ldidonato